### ISSUE TITLE
proper mnc2nii and nii2mnc pipelines

### ISSUE THREAD

**Issue by pbellec** [SIMEXP/niak#212]
The nii2mnc and nii2mnc bricks are actually a weird mixture of a brick and a pipeline. As a consquence, they behave weirdly when run in parallel, and don't have smart restart / updates capabilities. It would be (much) better to have a proper pipeline.

**Comment by pbellec** [comment]
too late to include in the cog release.

**Comment by pbellec** [comment]
Now that niak supports nifti, work on these tools is not a priority.
